### ISSUE TITLE
How to apply Ray Based Sampling to my own rendered images?

### ISSUE THREAD

**Issue by Zhangjzh** [kcyt/SPIFu#2]
Could you please tell me how to modified unroll_gt_smplx.py to apply Ray Based Sampling to my own rendered images. 
I modified unroll_gt_smplx.py by myself, but got some errors.
![error](https://user-images.githubusercontent.com/49779683/216814959-faacf48c-dfe2-47c2-bcef-20d5d5954608.png)
The shape of face_normals_average_np is [0, ] and the shape of features is [0, 4]. This shows that the num of faces that are hit is 0. I suspect this results from the difference of scale_factor. My scale_factor is 1.80061 rather than the original scale_factor 2.4851518.
I wonder what I should change if the scale_factor is different.
I would appreciate it a lot if you could give some suggestions! Thank you very mcuh!

**Comment by kcyt** [comment]
Indeed, it could be that in the code "gt_smplx_mesh.apply_transform(calib)", the 'calib' used is incorrect. You have to ensure that after "gt_smplx_mesh.apply_transform(calib)", the gt_smplx mesh must be within the range of [-1,1] in x, y, and z dimensions.
